src/pageObject.js

```diff
@@ -2,14 +2,38 @@
 
 class Header {
 
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
 };
 
 class ArticlePage extends PageObject {
+  constructor(url, slug) {
+    super(url);
+    this.url = url + slug;
```

Comment:
You're overwriting the `url` property of the parent `PageObject` class. This can lead to confusion and bugs. Instead, consider creating a new property, like `fullUrl`, to store the combination of `url` and `slug`.